app/routes/spin.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model() {
```

Comment:
This should use ember-data, and could, easily. https://guides.emberjs.com/v1.10.0/models/the-fixture-adapter/